.circleci/config.yml

```diff
@@ -110,7 +110,7 @@ jobs:
           command: WITH_LIBSNARK=1 RUSTFLAGS="-D warnings" ./integration_test.sh
   deploy:
     docker:
-      - image: circleci/python:latest-node
```

Comment:
This image was the cause of the broken zokrates.js builds, it is considered legacy now so I've changed it to a newer fixed version `cimg/python:3.8-node`